libs/ledger-live-common/src/device/use-cases/fetchMcusUseCase.ts

```diff
@@ -0,0 +1,8 @@
+import { fetchMcus } from "../../device-core/managerApi/use-cases/fetchMcus";
+import { HttpManagerApiRepositoryFactory } from "../factories/HttpManagerApiRepositoryFactory";
+
+export function fetchMcusUseCase(
+  managerApiRepository = HttpManagerApiRepositoryFactory.getInstance(),
```

Comment:
[COULD] write clearly the type of ManagerApiRepository

` managerApiRepository: ManagerApiRepository = HttpManagerApiRepositoryFactory.getInstance(),`